com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -110,25 +190,30 @@ private GlobalObjectId GetGlobalId()
             // If we can't get the asset GUID and/or the file identifier, then return the object identifier
             if (!AssetDatabase.TryGetGUIDAndLocalFileIdentifier(theAsset, out var guid, out long localFileId))
             {
-                Debug.Log($"[GlobalObjectId Gen][{theAsset.gameObject.name}] Failed to get GUID or the local file identifier. Returning default ({instanceGlobalId}).");
                 return instanceGlobalId;
             }
 
             // If we reached this point, then we are most likely opening a prefab to edit.            
             // The instanceGlobalId will be constructed as if it is a scene object, however when it
             // is serialized its value will be treated as a file asset (the "why" to the below code).
 
-            // Construct an imported asset identifier with the type being a source asset (type 3).
-            var prefabGlobalIdText = string.Format(k_GlobalIdTemplate, 3, guid, localFileId, 0);
+            // Object Types
+            // 0 = Null (we exit early on this type)
+            // 1 = Imported Asset
+            // 2 = Scene Object
+            // 3 = Source Asset.
+            var objetType = 3;
```

Comment:
Can these types be an enum something similar?

Reply:
Yeah constants would be good:

var objectType = k_GlobalObjectIdObjectType_SourceAsset;

would look much better, or a define or whatever :)